programs/mango-v4/src/accounts_ix/openbook_v2_place_order.rs

```diff
@@ -50,27 +52,23 @@ pub struct OpenbookV2PlaceOrder<'info> {
     pub event_heap: UncheckedAccount<'info>,
 
     #[account(mut)]
-    /// CHECK: base vault will be checked by openbook_v2
-    pub market_base_vault: Box<Account<'info, TokenAccount>>,
-
-    #[account(mut)]
-    /// CHECK: quote vault will be checked by openbook_v2
-    pub market_quote_vault: Box<Account<'info, TokenAccount>>,
+    /// CHECK: vault will be checked by openbook_v2
+    pub market_vault: Box<Account<'info, TokenAccount>>,
 
     /// CHECK: Validated by the openbook_v2 cpi call
     pub market_vault_signer: UncheckedAccount<'info>,
 
     /// The bank that pays for the order, if necessary
     // token_index and payer_bank.vault == payer_vault is validated inline at #3
     #[account(mut, has_one = group)]
-    pub payer_bank: AccountLoader<'info, Bank>,
+    pub bank: AccountLoader<'info, Bank>,
```

Comment:
There's been a bunch of pain around not having a writable payer_bank and receiver_bank in serum3_place_order. Strongly suggest passing payer_bank (mut) and receiver_bank (mut) explicitly. (and then it makes sense to keep the payer_ prefix again)

The oracles we should start reading from the health accounts instead, so those aren't essential and will start being ignored in the future.